core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -117,10 +119,26 @@ class BrokerToControllerChannelManager(metadataCache: kafka.server.MetadataCache
                                   callback: RequestCompletionHandler): Unit = {
     requestQueue.put(BrokerToControllerQueueItem(request, callback))
   }
+
+  private[server] def forwardRequest(requestBuilder: AbstractRequest.Builder[_ <: AbstractRequest],
+                                     responseToOriginalClient: (RequestChannel.Request, Int => AbstractResponse,
```

Comment:
This function has 3 callbacks... It would be nice if we could figure out how to pass through the `ForwardRequestHandler` directly.